### PR TITLE
Fix #5619: Fix Multiple Source Column Level Lineage

### LINKED ISSUE #5619: Column level lineage not getting generated
**Affected module**
Ingestion Framework

**Describe the bug**
Column Level not getting generated when there are multiple source table available in the query. for example:

```
CREATE VIEW view2 AS 
select product_id, product_name, model_year, list_price from production.products p join production.brands b on b.brand_id=p.brand_id;
```

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Fix #5619: Fix Multiple Source Column Level Lineage

#
### Type of change :

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/src/metadata/utils/sql_lineage.py [hunk 1]
```diff
@@ -233,21 +233,22 @@ def populate_column_lineage_map(raw_column_lineage):
...
         return lineage_map
     for source, target in raw_column_lineage:
-        if lineage_map.get(str(target.parent)):
-            ele = lineage_map.get(str(target.parent))
-            if ele.get(str(source.parent)):
-                ele[str(source.parent)].append(
-                    (
-                        target.raw_name,
-                        source.raw_name,
+        for parent in source._parent:
```
[the remark this answers is not in the document]
Comment (ulixius9) [reply ingestion/src/metadata/utils/sql_lineage.py:236]: Umm, will double check this 🤔

[the remark this answers is not in the document]
Comment (ulixius9) [reply ingestion/src/metadata/utils/sql_lineage.py:236]: So, I checked the library, if the column level lineage info is available the source._parent should have at least one parent   and `source._parent` is initialised with empty set so even if there are no parent it should be good to go

File: ingestion/src/metadata/utils/sql_lineage.py [hunk 2]
```diff
@@ -233,21 +233,22 @@ def populate_column_lineage_map(raw_column_lineage):
     if not raw_column_lineage or len(raw_column_lineage[0]) != 2:
```
Comment (ulixius9) [inline ingestion/src/metadata/utils/sql_lineage.py:233]: The column lineage info is verified with this condition @TeddyCr

**Review by TeddyCr — APPROVED** [review]
LGTM 🚀

**Pull request merged**